src/test/isolation2/output/resgroup/resgroup_startup_memory.source

```diff
@@ -0,0 +1,79 @@
+-- start_ignore
+! gpconfig -c runaway_detector_activation_percent -v 100;
+! gpstop -rai;
+-- end_ignore
+
+CREATE OR REPLACE FUNCTION resGroupPalloc(float) RETURNS int AS '/home/gpadmin/gpdb_src/src/test/isolation2/../regress/regress.so', 'resGroupPalloc' LANGUAGE C READS SQL DATA;
+CREATE
+
+ALTER RESOURCE GROUP default_group SET memory_limit 10;
+ALTER
+ALTER RESOURCE GROUP admin_group SET memory_limit 30;
+ALTER
+
+CREATE RESOURCE GROUP test_group with (concurrency=1, memory_limit=59, memory_spill_ratio=100, cpu_rate_limit=20);
+CREATE
+CREATE ROLE test_role RESOURCE GROUP test_group;
+CREATE
+CREATE ROLE default_role RESOURCE GROUP default_group;
+CREATE
+
+-- 1. Occupy all memory on a segment and fail with vmem tracker error
+ALTER RESOURCE GROUP admin_group SET concurrency 1;
+ALTER
+ALTER RESOURCE GROUP default_group SET concurrency 1;
+ALTER
+
+-- Start a session which will be detached from a group when the query is done
+-- resource groups can't see startup chunks occupied by a detached session but
+-- the vmem tracker can
+0: SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE pid != (SELECT pg_backend_pid());
```

Comment:
Why is this query required? What does exactly this test check?

Reply:
This test checks that the vmem tracker can see startup chunks of those processes which are not attached to any resource group. It starts the 0 session which terminates all other processes which can interfere this test's results and then creates a table just to make sure that this session is spawned on all segments. This session is then detached from a resource group and is tracked by vmem only.
Other sessions start transactions so that they can be attached to their resource groups. They allocate all available memory to a resource group. The last allocation should fail because of the vmem tracker, so we could make sure that the vmem tracker actually tracks the memory of those sessions which are not attached to a resource group

Reply:
What was the expected behavior before this patch?

Reply:
Bump .__.

Reply:
Actually, you are right here, this test is redundant. It supposed to be sort of sanity check, but since it behaves just like it would behave without the patch, it doesn't make sense, so I removed it